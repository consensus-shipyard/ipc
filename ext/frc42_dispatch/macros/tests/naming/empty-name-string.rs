use frc42_macros::method_hash;

fn main() {
	// this should panic due to empty string
	let _str_hash = method_hash!("");
}
