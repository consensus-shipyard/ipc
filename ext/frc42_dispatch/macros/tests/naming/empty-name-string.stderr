error: proc macro panicked
 --> tests/naming/empty-name-string.rs:5:18
  |
5 |     let _str_hash = method_hash!("");
  |                     ^^^^^^^^^^^^^^^^
  |
  = help: message: called `Result::unwrap()` on an `Err` value: EmptyString
